api/openapi/v2/system-agent.yaml

```diff
@@ -0,0 +1,625 @@
+        strategy:
+          type: string
+          description: "Indicates the expectation of whether a response should be produced synchronously or asynchronously. If asynchronously, desire for either a polling or push/callback should be provided. Valid values are 'sync','async-push','async-poll'"
+        type:
+          type: string
+          description: "The name of the type applicable to the request instance contained in the 'content' property."
+        version:
+          description: "Proposed field for explicitly defining version of request DTO. This is for advertising compatibility between a publisher/subscriber or requester/receiver"
+          type: string
+          example: "2.0.x"
+      required:
+        - action
+        - content
+        - strategy
+        - type
+        - version
+    ResponseEnvelope:
+      description: "A wrapper type for use when receiving a response from the /batch endpoint. Each individual response type in the HTTP response should be wrapped in an envelope to facilitate unmarshalling by the client. See property descriptions below for more details."
+      type: object
+      properties:
+        action:
+          type: string
+          description: "Indicates the type of operation applicable to the wrapped response. This should be recapitulated from the originating request. Valid values are 'create','read','update','delete', and 'command'"
+        content:
+          type: string
+          format: byte
+          description: "A byte array containing a marshalled response type instance. This is the specific, semantically identifiable response -- such as an AddDeviceResponse."
+        strategy:
+          type: string
+          description: "Recapitulates the expectation with regard to the delivery of response that was specified on the originating request. Valid values are 'sync','async-push','async-poll'"
+        type:
+          type: string
+          description: "The name of the type applicable to the response instance contained in the 'content' property."
+        version:
+          description: "Proposed field for explicitly defining version of response DTO. This is for advertising compatibility between a publisher/subscriber or requester/receiver"
+          type: string
+          example: "2.0.x"
+      required:
+        - action
+        - content
+        - strategy
+        - type
+        - version
+    SetConfigRequest:
+      description: "A request associated with the /config endpoint."
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      type: object
+      properties:
+        service:
+          description: "The service for which to obtain metrics"
+          type: string
+          example: "edgex-core-data"
+        configuration:
+          description: "Service's configuration"
+          type: object
```

Comment:
Since it'd be possible in v2 to send multiple use-case requests in a single transport request, I wonder if we still need to support requesting for more than one service in a use-case request.